website/docs/sequence.md

```diff
@@ -0,0 +1,430 @@
+The sequence diagram always knows only the currently available information and nothing on top.
+
+## Terminology
+
+In a sequence diagram, we have the following concepts:
+
+- Participant: A component that "participates" in the diagram, so something/someone whose behavior should be modeled.\
+  It can be one of the following:
+  - an `instance` without a name (`User`)
+  - an `instance` with a name (`Bob: User`)
+  - an `actor` (a stickman symbolizing a user), optionally with a name
+- Lifeline: the entire duration a participant is alive, symbolized by the dotted line downwards
+- Event: an x/y coordinate linking a point in time to a participant, accessed by using the syntax `event.participant`, so i.e. `startPayment.User`.\
+- Message: arrow between two `event`s with a semantic meaning
+- Activity: the time a participant is active
+- Frame: A box around some events. Can optionally contain a name (i.e. `if`, `while`), and sub compartments
+
+## Order matters
+
+Below, you'll find the order in which you should declare things so that they work as expected.\
+Hylimo walks through the diagram from left to right, and then from top to bottom.
+
+First, declare all participants (`instance`, `actor`) in the order you want to display them as they will be positioned on the x axis in this order:
+
+```hylimo
+sequenceDiagram {
+    instance("Bob")
+    instance("Shop")
+    actor("Admin")
+}
+```
+
+This example creates three participants, reading from left to right as `Bob`, `Shop` and lastly a user called `Admin`.
+
+Now that we've populated our `x` axis, let's move on with the `y` axis.
+
+The main way to move forward on the `y` axis in a sequence diagram is through `events`.\
+An `event` needs a name to refer to it later on and automatically moves downward on the y axis:
+
+```hylimo
+sequenceDiagram {
+    event("startPayment")
+}
+```
+
+As you can see above, the name should be understandable for you to know exactly which action is symbolized by this event.
+This name will never be shown anywhere.
+
+For most cases, the default value for how far away the next event will be good-enough.
+Nevertheless, in cases where you want an explicit distance to the predecessor, you are free to do so:
+
+```hylimo
+sequenceDiagram {
+    event("startPayment", 50)
+}
+```
+
+This positions the event `50` pixels below its predecessor.
+Note that while negative values are allowed here, they won't make any sense:
+Instead, you can also move the previous event up by that amount and invert their meanings.
+
+Both `event` and `participant` names will be registered as variables if they didn't exist already.
+In case they existed already, the existing name takes precedence.
+If their name is already used by something else, you can assign the result of these functions to a variable of your own choosing:
+
+```hylimo
+sequenceDiagram {
+    enableDebugging = true
+    Bob = true
+    user = instance("Bob")
+    event("buy")
+    event("stop")
+    buy.Bob --> stop.Bob
```

Comment:
It's hard to see where this arrow is, as there is also the dashed line in the background
maybe add styles (like color blue) or use a bidirectional arrow to highlight it a bit more?

Reply:
I've added now a manual axis alignment. In the future, we should compute one automatically.
However, I have no idea how to.